packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -502,9 +502,9 @@ export function f(items: any[]): any[] {
 }
 
 // [t]ext node
-export function t(text: string): VText {
-    const data = EmptyObject;
-    let sel, children, key, elm;
+export function t(text: string, data: CustomElementCompilerData): VText {
+    const { key } = data;
```

Comment:
Is the `data` a newly created param from the compiler OR is it pre-existing param that was not used? Will this break any existing pre-compiled component definitions?

Reply:
mm, good point, i left the VText to be [Key | undefined](https://github.com/salesforce/lwc/pull/1943#discussion_r444518663), trying to support the existing compilations, but i missed this case.

Though, we should probably dismiss existing pre-compiled components. thoughts?

Reply:
Yes, all the precompiled components have to be dismissed.